languages/csharp/exercises/concept/nullability/.docs/introduction.md

```diff
@@ -0,0 +1,93 @@
+Up to now, we saw how to declare variables and how to assign them to
+values.  For example, the following code snippet declares a variable
+`a` of type `int` whose value is `5`:
+
+```
+int a = 5;
+```
+
+Sometimes, we need to indicate that a variable has no specific value.s
+In C#, we can use the `null` literal to denote the abscence of a
+value.  A *nullable* type is a type that allow `null` values.  The
+operator `?` can be used to indicate that a type is nullable.
+
+In the following example, we declare a variable `b` of type `int?`
+whose value is `5`:
+
+```csharp
+int? b = 5;
```

Comment:
Maybe you can declare the nullable and non-nullable variable here at the same time?

```suggestion
int a = 4;
int? b = 5;
```